bin/kibana

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-NODE_OPTIONS="--no-warnings --max-http-header-size=65536 ${NODE_OPTIONS}" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
+NODE_OPTIONS="--max-http-header-size=65536 $(grep -v ^# < ${DIR}/config/node.options | xargs) $NODE_OPTIONS" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
```

Comment:
One issue here, is that `config` is a configurable location.

Reply:
@tylersmalley do you have any idea to get around this?

Reply:
@tylersmalley it looks like the only way to configure the config dir is using `process.env.KIBANA_PATH_CONF`. Do you think we can make it work by trying to read what is on `process.env.KIBANA_PATH_CONF or ${DIR}/config` ?

@jbudz how can we make this work for the `os_packages` being built with `fpm` ? Do you think we can just always export that `KIBANA_PATH_CONF` env var pointing to `/etc/kibana` ? I think that would do the trick and make it work everywhere.

Reply:
The only other way I know of is to pass `--config` to `bin/kibana` or `scripts/kibana`

Reply:
But the `--config` flag is for the file itself not for the folder right?

Reply:
yeah we can do that, good call.  we're not currently but we should be.

in the packages folder /etc/default/kibana adding `KIBANA_PATH_CONF` is the eventual.  if we can branch it to a separate PR that would be ideal,  mostly to keep the build times + qa distinct here.

Reply:
happy to take ^ above if we're okay with keeping it separate.  if it's a dependency on this PR and you want to do it, it's all yours too

Reply:
go ahead @jbudz

Reply:
@jbudz @tylersmalley I've just pushed a commit that fixes that problem and includes support for a configurable `config` folder in case the env var `KIBANA_PATH_CONF` is used. The only place it would not work right now is in the os_packages built with `fpm` but that would be solved when @jbudz put up the PR with the ideas we discuss above!